subsys/bluetooth/mesh/main.c

```diff
@@ -41,6 +41,8 @@
 #include "mesh.h"
 #include "gatt_cli.h"
 
+static bool is_initialized;
```

Comment:
This could be in `bt_mesh_net.flags`: https://github.com/zephyrproject-rtos/zephyr/blob/53e1ea58e04c445f314c70ced1c8aa6ea0b53ff8/subsys/bluetooth/mesh/net.h#L169-L185